open-craft/edx-platform
# fix: unhide discussion tab when enabling it

## Description

Update the `is_hidden` property in the modulestore to properly store the visibility of the discussions tab.

## Supporting information

- BB-9120

## Testing instructions

1. Create a course in a Palm devstack/sandbox.
2. Set the discussion tab to hidden.
3. Export the course.
4. Import the course in a Redwood devstack/sandbox.
5. Create a Course Update
6. Verify the discussion tab disappears in the LMS.
7. Apply this change to the devstack/sandbox.
8. Enable the discussions in Studio Pages & Resources.
9. Create a Course Update.
10. Verify the discussion tab does not disappear this time.

## Deadline

ASAP
